src/raftstore/store/util.rs

```diff
@@ -849,8 +878,8 @@ mod tests {
         let db_opts = DBOptions::new();
         let mut cf_opts = ColumnFamilyOptions::new();
         cf_opts.set_level_zero_file_num_compaction_trigger(10);
-        let f = Box::new(SizePropertiesCollectorFactory::default());
-        cf_opts.add_table_properties_collector_factory("tikv.size-collector", f);
+        let f = Box::new(RangePropertiesCollectorFactory::default());
```

Comment:
do we have a test that there is no range-collector and we fall back to the size collector?

Reply:
Yes, We've tested it manually. This case only happens when we update size-collector to range-collector, and quickly fall back to size-collector. Then for the range may have the following states:
1. have both `size-property` and `range-property`, the size comes from range-property, it works well.
2. have `size-property`, no `range-property`, the size comes from size-properties, it works well.
3. no `size-property`, has `range-property`, it works as state 1
4. no `size-property`, no `range-property`, the size may be smaller than the actual value but not 0, the merge operation may be called by `PD`, we need to close it.

Reply:
@AndreMouche 

no manully, we need a unit test for it.

Reply:
yes, we already have such unit test.